sbin/ipf/ipf/ipf.8

```diff
@@ -158,7 +158,7 @@ Zero global statistics held in the kernel for filtering only (this doesn't
 affect fragment or state statistics).
 .DT
 .SH ENVIRONMENT
-.NM utilizes the following environment variable.
+ipf utilizes the following environment variable.
```

Comment:
There's no `.NM` in man(7).  The author was clearly thinking of mandoc; for plain old man, we have to write out the name.

Reply:
Sorry for the noise, I was confused because mandoc(1) is the compiler which speaks both man(7) and mdoc(7) roff(7) dialects (plus some others like tbl(7), roff(7) is an amazing language).

Reply:
I haven't seen your "earlier deleted comment", but this comment made me realize that I'd written "mandoc" instead of "mdoc(7)".

I've edited my earlier comment; hopefully it makes more sense now.